docker/colore/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ruby:2.6.5
```

Comment:
This is using Debian

There is a centos 7 official image with Ruby 2.6, but it is not locked to 2.6.5

Ref: 
- https://hub.docker.com/layers/centos/ruby-26-centos7/latest/images/sha256-3827a0a4f348a3f9815d1f8d123eafbe129cdf57c7f8e0ed6265270d286cae80?context=explore
- https://hub.docker.com/_/ruby?tab=description&page=1&ordering=last_updated